### PR TITLE
Reformat code (2)

### PR DESCRIPTION
**Pull request by anthrotype**
let's work on #413 together on a shared branch on the upstream repo

### CODE REVIEW

**Comment by belluzj** [comment]
Before you blacken everything, would you want to review and merge the two other open PRs?
* https://github.com/googlei18n/glyphsLib/pull/405
* https://github.com/googlei18n/glyphsLib/pull/407

I'm afraid they might have conflicts after that, and your blackening thing is all automated right?

**Comment by anthrotype** [comment]
@belluzj good point Jany. I'm just setting things up so that we are ready to reformat everything. I'll wait for the other unfinished stuff before merging this.

**Comment by anthrotype** [comment]
I also added flake8 and started fixing some stuff but the list is too long, will finish in next days

File: Lib/glyphsLib/builder/font.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def to_ufo_font_attributes(self, family_name):
     designer_url = font.designerURL
     manufacturer = font.manufacturer
     manufacturer_url = font.manufacturerURL
-    note = font.note
+    # XXX note is unused?
+    # note = font.note
```
Comment (madig) [inline Lib/glyphsLib/builder/font.py]: Note to self: test this

File: tests/writer_test.py [hunk 2]
```diff
@@ -780,7 +780,8 @@ def test_write_layer(self):
         # bounds: read-only, computed
         # selectionBounds: read-only, computed
         # background
-        bg = layer.background
+        # XXX bg is unused?
```
Comment (madig) [inline tests/writer_test.py]: Note to self: Investigate.

**Comment by madig** [comment]
PyCharm has even more checks. Let's run a few.

File: Lib/glyphsLib/classes.py [hunk 3]
```diff
@@ -65,6 +65,7 @@
     "GSHint",
     "GSBackgroundImage",
     # Constants
+    "__all__",
```
Comment (anthrotype) [inline Lib/glyphsLib/classes.py]: Not sure adding __all__ to __all__ is a good idea..

**Comment by madig** [comment]
Oh boy... there are quite a few warnings about attributes being defined outside `__init__`, e.g. https://github.com/googlei18n/glyphsLib/blob/master/Lib/glyphsLib/classes.py#L1338

**Comment by anthrotype** [comment]
We can’t fix everything now. Let’s start with making flake8 pass

**Comment by madig** [comment]
Ok, so I'll stop with PyCharm now :) The rest looks okay to me.

**Pull request merged**